jupyter/muehle-game.ipynb

```diff
@@ -236,11 +236,11 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "def finished(s):\n",
-    "    return not (hasEnoughStones(s, 'w') and \\\n",
-    "                hasEnoughStones(s, 'b') and \\\n",
-    "                len(nextStates(s, 'w')) > 0 and \\\n",
-    "                len(nextStates(s, 'b')) > 0)"
+    "def finished(s, p):\n",
+    "    return not hasEnoughStones(s, 'w') or \\\n",
+    "           not hasEnoughStones(s, 'b') or \\\n",
+    "           (p == 'w' and len(nextStates(s, 'w')) == 0) or \\\n",
+    "           (p == 'b' and len(nextStates(s, 'b')) == 0)"
```

Comment:
Ist die erste Abfrage (p == 'w') überhaupt notwendig? Ist das Spiel nicht so oder so vorbei, wenn ein Spieler keine nextStates mehr hat, egal welche Farbe es ist?

Reply:
Ne, das war ja genau der Bug: Ich hatte glaube ich sogar ein Beispiel in #39

Reply:
Ah okay, makes sense